com.ibm.wala.cast.js.rhino/src/test/java/com/ibm/wala/cast/js/rhino/callgraph/fieldbased/test/AbstractFieldBasedTest.java

```diff
@@ -58,6 +64,29 @@ protected JSCallGraph runTest(URL url, Object[][] assertions, BuilderType... bui
     return cg;
   }
 
+  protected JSCallGraph runBoundedTest(
+      String script, Object[][] assertions, BuilderType builderType, int bound)
+      throws WalaException, Error, CancelException {
+    JSCallGraph cg = null;
+    // URL url = TestFieldBasedCG.class.getClassLoader().getResource(script);
+    JavaScriptLoaderFactory loaders = new JavaScriptLoaderFactory(new CAstRhinoTranslatorFactory());
+    IProgressMonitor monitor = ProgressMaster.make(new NullProgressMonitor(), 45000, true);
+    List<Module> scripts = new ArrayList<>();
+    URL url = TestFieldBasedCG.class.getClassLoader().getResource(script);
+    scripts.add(new SourceURLModule(url));
+    scripts.add(JSCallGraphUtil.getPrologueFile("prologue.js"));
+    try {
+      cg =
+          util.buildBoundedCG(loaders, scripts.toArray(new Module[0]), monitor, false, bound)
+              .getCallGraph();
+      System.err.println(cg);
+      verifyGraphAssertions(cg, assertions);
+    } catch (AssertionError afe) {
+      throw new AssertionError(builderType + ": " + afe.getMessage());
```

Comment:
```suggestion
      throw new AssertionError(builderType + ": " + afe.getMessage(), afe);
```
This change should give you the full call stack for the original error which can be helpful for debugging